### ISSUE TITLE
Is Altstore Jit enabled supported on older macbook air (2014)? Error on iphone ios 17.2.1

### ISSUE THREAD

**Issue by yugioh2king** [altstoreio/AltStore#1374]
I am getting an error on my macbook air (2014) when I enable JIT on my iphone xs max (ios 17.2.1) it days JIT could not be enabled for DolphinIOS. The URL to download the Developer disk image could not be determined.

Does anyone have a fix for this?

**Comment by shanegillio** [comment]
To use JIT on iOS 17, you need to install extra tools: https://faq.altstore.io/how-to-use-altstore/altjit

**Comment by yugioh2king** [comment]
I checked the link that you gave me, but it says "If you are running macOS Sonoma and see an error that Command Line Tools aren’t currently available, try the following commands instead:". 

It does not give me instructions for older mac os version.

I do not have Mac OS Sonoma. My Macbook is from early 2014 and the lastest mac os version is big sur and I cannot update to the latest Mac OS version. This macbook cannot update to Sonoma.

**Comment by SKI79** [comment]
You can still try to run the commands, but if they don't work, you might need a newer Mac.

**Comment by yugioh2king** [comment]
> You can still try to run the commands, but if they don't work, you might need a newer Mac.

I've tried to run the script like you said and it unfortunately does not work on big sur. Oh well thanks anyway.

**Comment by kof117** [comment]
An alternate method could be to use something like OpenCore Legacy Patcher and run newer macOS on an older computer, but I don't know how well if at all that will work.

**Comment by yugioh2king** [comment]
> An alternate method could be to use something like OpenCore Legacy Patcher and run newer macOS on an older computer, but I don't know how well if at all that will work.

Yes I am aware of that and seen videos on youtube on how to install that, however I don't want to make a mistake trying to install a latest os of the mac and accidentally damage it. I will be patient and get a better macbook that supports Mac OS Sonoma.

Just want to say thanks for the info and help.
